apps/examples/libcoap_client/libcoap-client.c

```diff
@@ -363,11 +363,11 @@ static inline coap_opt_t *get_block(coap_pdu_t *pdu, coap_opt_iterator_t *opt_it
 	return coap_option_next(opt_iter);
 }
 
-#define HANDLE_BLOCK1(Pdu)						\
-  ((method == COAP_REQUEST_PUT || method == COAP_REQUEST_POST) &&	\
-   ((flags & FLAGS_BLOCK) == 0) &&					\
-   ((Pdu)->hdr->code == COAP_RESPONSE_CODE(201) ||			\
-    (Pdu)->hdr->code == COAP_RESPONSE_CODE(204)))
+#define HANDLE_BLOCK1(Pdu)											\
+	((method == COAP_REQUEST_PUT || method == COAP_REQUEST_POST) &&	\
+	 ((flags & FLAGS_BLOCK) == 0) &&								\
+	  ((Pdu)->hdr->code == COAP_RESPONSE_CODE(201) ||				\
+	   (Pdu)->hdr->code == COAP_RESPONSE_CODE(204)))
```

Comment:
Here are spaces

Reply:
Our coding rule : "please, no spaces at the start of a line" means
<Spaces>              ->ERROR
<Spaces><Tabs> -> ERROR
<Tabs>                 ->OK
<Tabs><Spaces> -> OK

Reply:
I know. But why do you add spaces at there? for what?